ctapipe/ml/tools/train_disp_reconstructor.py

```diff
@@ -0,0 +1,210 @@
+import astropy.units as u
+import numpy as np
+
+from ctapipe.coordinates.disp import horizontal_to_telescope
+from ctapipe.core import Tool
+from ctapipe.core.traits import Bool, Int, Path
+from ctapipe.io import EventSource, TableLoader
+
+from ..apply import CrossValidator, DispClassifier, DispRegressor
+from ..preprocessing import check_valid_rows
+from ..sklearn import Classifier, Regressor
+
+
+def euclidean_distance(x1, y1, x2, y2):
+    return np.sqrt((x1 - x2) ** 2 + (y1 - y2) ** 2)
+
+
+class TrainDispReconstructor(Tool):
+    """Train two ML models for origin reconstruction using the disp method"""
+
+    name = "ctapipe-train-disp"
+    description = __doc__
+
+    output_path_reg = Path(
+        default_value=None, allow_none=False, directory_ok=False
+    ).tag(config=True)
+
+    output_path_clf = Path(
+        default_value=None, allow_none=False, directory_ok=False
+    ).tag(config=True)
+
+    n_events = Int(default_value=None, allow_none=True).tag(config=True)
+    random_seed = Int(default_value=0).tag(config=True)
+    project_disp = Bool(default_value=False).tag(config=True)
```

Comment:
this needs the `help` argument set

Reply:
Do you mean `help` should be added to `n_events` and `random_seed`, too?

Reply:
no, those are clear by the name. I have just no idea what `project_disp` does.